kratos/utilities/divide_triangle_2d_3.h

```diff
@@ -46,15 +46,15 @@ class KRATOS_API(KRATOS_CORE) DivideTriangle2D3 : public DivideGeometry
     ///@name Type Definitions
     ///@{
 
-    typedef DivideGeometry                                                    BaseType;
-    typedef typename BaseType::GeometryType                               GeometryType;
-    typedef typename BaseType::IndexedPointType                       IndexedPointType;
-    typedef Line2D2 < IndexedPointType >                          IndexedPointLineType;
-    typedef Triangle2D3 < IndexedPointType >                  IndexedPointTriangleType;
+    typedef DivideGeometry                          BaseType;
+    typedef BaseType::GeometryType                  GeometryType;
+    typedef BaseType::IndexedPointType              IndexedPointType;
+    typedef Line2D2 < IndexedPointType >            IndexedPointLineType;
+    typedef Triangle2D3 < IndexedPointType >        IndexedPointTriangleType;
 
     /// Pointer definition of DivideTriangle2D3
     KRATOS_CLASS_POINTER_DEFINITION(DivideTriangle2D3);
-    
+
     const std::vector<int> mEdgeNodeI = {0, 1, 2};
```

Comment:
out of scope, but it just came to my mind that a good solution would be to use std::array instead of std::vector here.

feel free to ignore this comment, it is just for the records

Reply:
I'll check it out.